virtualizarr/tests/test_readers/conftest.py

```diff
@@ -387,7 +387,7 @@ def scalar_fill_value_hdf5_file(tmpdir):
 
 @pytest.fixture(params=fill_values)
 def cf_fill_value_hdf5_file(tmpdir, request):
-    filepath = tmp_path / f"cf_fill_value_{request.param['label']}.nc"
+    filepath = tmpdir / f"cf_fill_value_{request.param['label']}.nc"
```

Comment:
You can keep `tmpdir`, but pytest has replaced it with `tmp_path`, so it's a typo only if you don't also change the name of the function parameter :-)

Reply:
I'll leave this for later, just hoping to get the tests all green before heading out for the weekend :)